### PR TITLE
[NUI] Make some of Progress and Slider public methods

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Description of Change ###
Public open API list
- Progress : IndeterminateImageUrl
- ProgressStyle : IndeterminateImageUrl
- Slider : ThumbImageUrl, IsValueShown, ValueIndicatorText,
           ValueIndicatorSize, ValueIndicatorUrl, IsDiscrete, DiscreteValue
- SliderStyle : ValueIndicatorText, ValueIndicatorImage

### API Changes ###
 - ACR: TCSACR-408
 - https://code.sec.samsung.net/jira/browse/TCSACR-408

### CODE REVIEW

**Review by hyunjushin — COMMENTED** [review]
The naming of Warningxxx in Slider is not natural. Can you explain more in detail?

**Comment by Seoyeon2Kim** [comment]
> The naming of Warningxxx in Slider is not natural. Can you explain more in detail?

There's a track for warning alarm in Slider UX guide.
It's like a section where, when you listen to music, if you turn it up more a certain level of volume, slider thumb turns red, and so on.

**Comment by Seoyeon2Kim** [comment]
'Warningxxx' properties are not opened in this version.

**Comment by myroot** [comment]
> 
> 
> > The naming of Warningxxx in Slider is not natural. Can you explain more in detail?
> 
> There's a track for warning alarm in Slider UX guide.
> It's like a section where, when you listen to music, if you turn it up more a certain level of volume, slider thumb turns red, and so on.

it is too specific naming and feature to use on common slider API, 

in my opinion, Slider API only provide a way to change slider background bar. 
and App use it to implement music app UX,

**Review by hyunjushin — APPROVED** [review]

**Comment by Seoyeon2Kim** [comment]
> > > The naming of Warningxxx in Slider is not natural. Can you explain more in detail?
> > 
> > 
> > There's a track for warning alarm in Slider UX guide.
> > It's like a section where, when you listen to music, if you turn it up more a certain level of volume, slider thumb turns red, and so on.
> 
> it is too specific naming and feature to use on common slider API,
> 
> in my opinion, Slider API only provide a way to change slider background bar.
> and App use it to implement music app UX,

Okay, thank you for your good opinion.
I'm not going to public open for warning area.

File: src/Tizen.NUI.Components/Controls/Slider.cs [hunk 1]
```diff
@@ -926,8 +920,7 @@ public string ValueIndicatorUrl
         /// Then, the thumb can only go to 0, 20, 40, 60, 80, and 100.
         /// The default is 0.
         /// </summary>
-        /// This will be public opened later after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```
Comment (wiertel) [inline src/Tizen.NUI.Components/Controls/Slider.cs:947]: The detailed description would better fit a \<details\> section. I mean I would suggest splitting the summary: leave first 2 sentences in \<summary\>, put the rest in \<details\>.

Comment (Seoyeon2Kim) [reply src/Tizen.NUI.Components/Controls/Slider.cs:947]: I update this in another PR : https://github.com/Samsung/TizenFX/pull/2990

**Pull request merged**